src/MirakaiScrollsRenderer.sol

```diff
@@ -71,16 +94,15 @@ contract MirakaiScrollsRenderer is Ownable {
                 abi.encodePacked(
                     "data:application/json;base64,",
                     Base64.encode(
-                        bytes(
-                            abi.encodePacked(
-                                '{"name": "Scroll',
-                                toString(tokenId),
-                                '", "description": "description x", "image": "data:image/svg+xml;base64,',
-                                render(tokenId, dna),
-                                '", "attributes":',
-                                formatTraits(traitIndexes),
-                                "}"
-                            )
+                        abi.encodePacked(
+                            '{"'
+                                '"name": "Scroll ', toString(tokenId), '",'
```

Comment:
@Saw-mon-and-Natalie  I believe there's an extra quote in front of name making the json invalid

Reply:
You are right. Pushed a fix. Great catch.